src/replay.ts

```diff
@@ -68,13 +68,17 @@ async function run() {
   const messages = await getNextMessages(lastMci);
 
   if (messages && messages.length > 0) {
-    // Process messages
-    await processMessages(messages);
+    try {
+      // Process messages
+      await processMessages(messages);
```

Comment:
Isnt there a possibility that we would send the same notification in loop? If the 2nd process message fail, it will go over the first one again and again no?